docs/spec/relayer-v2/Chain.tla

```diff
@@ -0,0 +1,146 @@
+---------------------------- MODULE Chain ----------------------------
+
+EXTENDS Naturals, FiniteSets, RelayerDefinitions, 
+        ClientHandlers, ConnectionHandlers, ChannelHandlers
+        
+CONSTANTS ChainID, \* chain identifier
+          Heights \* set of possible heights of the chains in the system 
+
+VARIABLES chainStore,
+          incomingDatagrams
+
+vars == <<chainStore, incomingDatagrams>>          
+
+(***************************************************************************
+ Client update operators
+ ***************************************************************************)
+\* Update the clients on chain with chainID, 
+\* using the client datagrams generated by the relayer      
+\* (Handler operators defined in ClientHandlers.tla)
+LightClientUpdate(chainID, chain, datagrams) == 
```

Comment:
Is second parameter chain or chainStore?

Reply:
`chainStore` is an already defined variable, so there's a name clash. I can rename the variable to `store`, if that's more intuitive.